.github/workflows/release.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       contents: read
       packages: write
     env:
-      IMAGE_NAME: autoinstrumentation-go
+      IMAGE_NAME: otel-autoinstrumentation-go
```

Comment:
This will make the docker image `otel/otel-autoinstrumentation-go`. Do we want the stutter in the name?

Reply:
Hmm, no I don't think the additional `otel` prefix.